utils/bin/lvim

```diff
@@ -3,4 +3,36 @@
 export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-$HOME/.local/share/lunarvim}"
 export LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-$HOME/.config/lvim}"
 
+headless_update() {
+  nvim --cmd 'lua _G.__lvim_updating=true' -u "$LUNARVIM_RUNTIME_DIR/lvim/init.lua" --noplugin -n -i NONE --headless
+}
+
+for arg; do
+  shift
+  case $arg in
+    --update)
+      :
+      echo 'Updating LunarVim...'
+      old="$(pwd)"
+      cd "$LUNARVIM_RUNTIME_DIR/lvim"
+      target_branch="${1:-$(git rev-parse --abbrev-ref HEAD)}"
```

Comment:
If it doesn't quit, then there's a problem here because you're expecting the name of the branch as a positional argument, but also assigning a default one, so any arguments that are passed after will be passed to `git fetch` without sanitization.

```suggestion
      target_branch="${LV_BRANCH:-$(git rev-parse --abbrev-ref HEAD)}"
```

Reply:
It does quit.
Not expecting a branch, it is optional. If a branch is provided it will fetch and use it, otherwise it uses the current branch.
If it fails, the script also fails.